admin_app_2/src/utils/api.ts

```diff
@@ -0,0 +1,126 @@
+// Temp read bearer token from file. Will be removed when auth is implemented.
+const json = require("../../temp_secrets.json");
+const ACCESS_TOKEN = json.ACCESS_TOKEN;
+const BACKEND_ROOT_PATH = "http://localhost:8000";
+
+interface ContentBody {
+  content_title: string;
+  content_text: string;
+  content_language: string;
+  content_metadata: Record<string, unknown>;
+}
+
+const getContentList = async () => {
+  return fetch(`${BACKEND_ROOT_PATH}/content/list`, {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${ACCESS_TOKEN}`,
+    },
+  }).then((response) => {
+    if (response.ok) {
+      let resp = response.json();
+      return resp;
+    } else {
+      throw new Error("Error fetching content list");
+    }
+  });
+};
+
+const getContent = async (content_id: number) => {
+  return fetch(`${BACKEND_ROOT_PATH}/content/${content_id}`, {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${ACCESS_TOKEN}`,
+    },
+  }).then((response) => {
+    if (response.ok) {
+      let resp = response.json();
+      return resp;
+    } else {
+      throw new Error("Error fetching content");
+    }
+  });
+};
+
+const deleteContent = async (content_id: number) => {
```

Comment:
content_id is a string everywhere else. VSCode typing complains

```suggestion
const deleteContent = async (content_id: string) => {
```